File: .claude/skills/database-architecture/examples/javascript/sequelize/crud_example.ts
import { Sequelize, DataTypes } from 'sequelize'

const sequelize = new Sequelize({
  dialect: 'sqlite',
  storage: 'demo.db',
  logging: false,
})

const User = sequelize.define('User', {
  id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
  name: { type: DataTypes.STRING, allowNull: false },
  email: { type: DataTypes.STRING, allowNull: false, unique: true },
})

await sequelize.sync()

const user = await User.create({ name: 'John', email: 'john@example.com' })
const users = await User.findAll()
console.log(users)

await User.destroy({ where: { id: user.id } })

File: .claude/skills/database-architecture/examples/python/sqlalchemy/crud_example.py
from sqlalchemy import Column, Integer, String, create_engine
from sqlalchemy.orm import sessionmaker, declarative_base

Base = declarative_base()

class User(Base):
    __tablename__ = 'users'
    id = Column(Integer, primary_key=True, autoincrement=True)
    name = Column(String(100))
    email = Column(String(255), unique=True)

engine = create_engine('sqlite:///demo.db')
Base.metadata.create_all(engine)
Session = sessionmaker(bind=engine)

session = Session()

session.add(User(name='John', email='john@example.com'))
session.commit()

users = session.query(User).all()
print(users)

session.query(User).filter(User.name == 'John').delete()
session.commit()